OrderApi/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ShoesOnContainers.Services.OrderApi.Data;
+using Microsoft.EntityFrameworkCore;
+using ShoesOnContainers.Services.OrderApi.Models;
+using Microsoft.Extensions.Logging;
+using System.Net;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Options;
+using MassTransit;
+using Common.Messaging;
+
+namespace ShoesOnContainers.Services.OrderApi.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [Authorize]
+     
+    public class OrdersController : Controller
+    {
+
+        private readonly OrdersContext _ordersContext;
+      private readonly IOptionsSnapshot<OrderSettings> _settings;
```

Comment:
do we want to use IOptionsSnapshot or use IConfiguration injection like she ended up using?
```suggestion
      private readonly IConfiguration _config;
```